api/control/v1alpha1/register.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package v1alpha1
 
 import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
why do we need `k8sruntime` alias?

Reply:
It's not the only "runtime" in our world.